src/App.tsx

```diff
@@ -1,61 +1,40 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { useState } from 'react';
+import { Todo } from './types/Todo';
+import { getUserById } from './services/user';
+import { TodoForm } from './components/TodoForm/TodoForm';
+
+function getNewTodoId(todos: Todo[]) {
+  const maxId = Math.max(...todos.map(todo => todo.id));
+
+  return maxId + 1;
+}
+
+export const initialTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
```

Comment:
The `getUserById` function is used here to assign a user to each todo. This is correct according to the checklist item 5, which states that data should be prepared in one place (App component) and passed to child components. However, ensure that `getUserById` does not return `null` or `undefined` values, as per checklist item 4, which advises against rendering components with `null` or `undefined` values.